lib/new-admin/graphql/schema.js

```diff
@@ -159,12 +187,14 @@ const typeDefs = gql`
     cryptoCurrencies: [CryptoCurrency]
     machines: [Machine]
     customers: [Customer]
+    customer(customerId: ID!): Customer
     machineLogs(deviceId: ID!): [MachineLog]
     funding: [CoinFunds]
     serverVersion: String!
     uptime: [ProcessStatus]
     serverLogs: [ServerLog]
     transactions: [Transaction]
+    customerTransactions(customerId: ID!): [Transaction]
```

Comment:
I think this should be a field under customer with a custom resolver.

Reply:
Didn't know it was possible, but agreed, waaaaay better